Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -574,7 +592,11 @@ - (void)endSearching:(id)sender {
     
     self.searching = NO;
 
-    [self.navigationController setToolbarHidden:YES animated:YES];
+    if (UIDevice.isPad) {
+        [self.navigationController setToolbarHidden:YES animated:YES];
+    } else {
+        [self configureNavigationControllerToolbar];
+    }
```

Comment:
Note: You can probably encapsulate this logic in the `configureNavigationControllerToolbar` API 

```
guard !UIDevice.isPad else {
   navigationController.setToolbarHidden(true, animated: true)
   return
}
```

Reply:
Ah cool idea.  There was still some work being done in configureNavigationControllerToolbar that other times needed to be run, but was able to put the display/hide into that work like this:

if (UIDevice.isPad) {
        [self configureSearchToolbar];
        [self.navigationController setToolbarHidden:!self.searching animated:YES];
        return;
    }

which works great.  Thanks!